android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -173,6 +174,17 @@ class DVCClient private constructor(
         }
     }
 
+    fun close(callback: DVCCallback<String>? = null) {
```

Comment:
any tests that can be written for this? I wrote a test on iOS you could refer to

Reply:
yeah, ill get on that

Reply:
pushed up an attempt at a unit test, but it seems weird that were doing assertions on logs existing